commit 15bf041ebd6386e81db7cbf1a65637b2d79928d4
Author: James Hughes <james.hughes@raspberrypi.org>
Date:   Wed Oct 31 13:00:46 2018 +0000

    Increase firmware call buffer size to 48 bytes
    
    An assumption was made in commit a1547e0bc that 32 bytes
    would be enough data buffer size for all firmware calls. However,
    the axi performance monitor driver uses a call with 44 bytes
    (RPI_FIRMWARE_GET_PERIPH_REG) to get the VC registers values.
    
    Increase value to 48 to take this in to account.
    
    Signed-off-by: James Hughes <james.hughes@raspberrypi.org>

diff --git a/drivers/firmware/raspberrypi.c b/drivers/firmware/raspberrypi.c
index dc95fa38e5d6..a4225b74f154 100644
--- a/drivers/firmware/raspberrypi.c
+++ b/drivers/firmware/raspberrypi.c
@@ -22,7 +22,7 @@
 #define MBOX_DATA28(msg)		((msg) & ~0xf)
 #define MBOX_CHAN_PROPERTY		8
 
-#define MAX_RPI_FW_PROP_BUF_SIZE	32
+#define MAX_RPI_FW_PROP_BUF_SIZE	48
 
 static struct platform_device *rpi_hwmon;
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1250_15bf041ebd6386e81db7cbf1a65637b2d79928d4.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1250_15bf041ebd6386e81db7cbf1a65637b2d79928d4.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
